test/lib/analyzer/java.spec.ts

```diff
@@ -70,35 +67,25 @@ describe("parsePomProperties function", () => {
       const fixture = getTextFromFixture(
         "pom-properties/invalid.pom.properties",
       );
-      const path = "/path/to/org.test.dependency-1.0.0.jar";
-      const dep = getDependencyFromPomProperties(fixture, path);
-      expect(dep).toBeNull();
+      const coords = getCoordsFromPomProperties(fixture);
+      expect(coords).toBeNull();
     });
   });
 });
 
-describe("getDependencyFromPomProperties function", () => {
+describe("getCoordsFromPomProperties function", () => {
   const fixture = getTextFromFixture("pom-properties/valid.pom.properties");
-  const path = "/path/to/package-dependency-1.0.0.jar";
-  const dep = parsePomProperties(fixture);
-
-  it("returns a dep as expected", () => {
-    const dep = getDependencyFromPomProperties(fixture, path);
-    expect(dep).not.toBeNull();
-  });
 
-  it("returns null when the dependency references the JAR", () => {
-    const path = "/path/to/org.test.dependency-1.0.0.jar";
-    const dep = getDependencyFromPomProperties(fixture, path);
-    expect(dep).toBeNull();
+  it("returns a coord as expected", () => {
+    const coords = getCoordsFromPomProperties(fixture);
+    expect(coords).not.toBeNull();
```

Comment:
would it be useful to check the coords are valid?

Reply:
`parsePomProperties` has a test for invalid coords so not needed here I think.